src/frontend/magnify/packages/components/src/components/rooms/RoomConfig/Users/RoomUsersConfig.test.tsx

```diff
@@ -82,12 +71,19 @@ describe('RoomUsersConfig', () => {
     expect(baseElement.querySelector('#add-room-user')).not.toBeInTheDocument();
 
     const addMember = await screen.findByRole('button', { name: 'Add a member' });
-    await act(async () => {
-      await user.click(addMember);
+    await act(() => {
+      user.click(addMember);
     });
 
     await waitFor(async () => {
       expect(baseElement.querySelector('#add-room-user')).toBeInTheDocument();
     });
   });
 });
+
+export async function resolveAll<T>(
```

Comment:
Put it in a separate file as it could be used elsewhere than in this file

Reply:
true